cmd/job-executor-service-initcontainer/main.go

```diff
@@ -63,6 +68,7 @@ func main() {
 	fs := afero.NewOsFs()
 
 	baseURL, err := url.Parse(env.KeptnAPIURL)
+	log.Printf("BaseURL casting: %s", baseURL)
```

Comment:
Is this `log.Printf` a thing left from debugging code, or is it intended for the output? If so I would prefer to have a different message, like "Keptn API URL: %s"

Reply:
Yes, that was a debug statement and should be removed.